### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by Sohelhere** [community/community#34973] [New to GitHub] [answered]
Hii! I am new to GitHub and need some help like how to contribute my code ,how work with repositories and how to contribute in other repositories

**Comment by Ba4bes** [comment]
Hi!
Maybe GitHub Skills can help you out: <https://skills.github.com/>
There are a few practical lessons to get you started. [This one might be a nice one to begin with](https://github.com/skills/introduction-to-github)

**Reply by Sohelhere** [reply]
HELLO there!we know each other on GitHub
Thank for your help
But till branch i have done while making new respiratory i am coming across some issue would you help me

**Comment by MelphinDev** [comment, marked as answer]
This repository have instructions on how to contribute.
https://github.com/firstcontributions/first-contributions

**Reply by emmanuelladeku48-cell** [reply]
please i can't do it
